binding/python/setup.py

```diff
@@ -0,0 +1,64 @@
+#
+# Copyright 2024 Picovoice Inc.
+#
+# You may not use this file except in compliance with the license. A copy of the license is located in the "LICENSE"
+# file accompanying this source.
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+# an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+#
+
+import os
+import shutil
+
+import setuptools
+
+INCLUDE_FILES = ('../../LICENSE', '__init__.py', '_factory.py', '_orca.py', '_util.py')
+INCLUDE_LIBS = ('common', 'jetson', 'linux', 'mac', 'raspberry-pi', 'windows')
+
+os.system('git clean -dfx')
+
+package_folder = os.path.join(os.path.dirname(__file__), 'pvorca')
+os.mkdir(package_folder)
+manifest_in = ""
+
+for rel_path in INCLUDE_FILES:
+    shutil.copy(os.path.join(os.path.dirname(__file__), rel_path), package_folder)
+    manifest_in += "include pvorca/%s\n" % os.path.basename(rel_path)
+
+os.mkdir(os.path.join(package_folder, 'lib'))
+for platform in INCLUDE_LIBS:
+    shutil.copytree(
+        os.path.join(os.path.dirname(__file__), '../../lib', platform),
+        os.path.join(package_folder, 'lib', platform))
+manifest_in += "recursive-include pvorca/lib *\n"
```

Comment:
correct me if I'm wrong but this will include both the male and female models, but above it looks like only the female model is to the user? If this is the case, we should only ship one model or build in the ability for the user to select between male and female default models somehow

Reply:
See above. It's handled via the init function. I will also add it to the READMEs.

Reply:
I'm not clear on how this is handled via init - it still seems as though we are shipping both models, but only providing one as a default model in code. We should either ship a single model or offer the user the ability to choose between the two default models we ship.